middle_end/flambda/inline_and_simplify.ml

```diff
@@ -800,7 +800,7 @@ and simplify_partial_application env r ~lhs_of_application
     Location.prerr_warning (Debuginfo.to_location dbg)
       (Warnings.Inlining_impossible "[@unroll] attributes may not be used \
```

Comment:
Is the same `[@unroll]` -> `[@unrolled]` change from `flambda_middle_end.ml` needed here too?